src/components/Collapser.js

```diff
@@ -44,7 +44,6 @@ class Collapser extends React.Component {
                 onclick={this.handleClick}
             >
                 <a
-                    href="#"
                     alt="Show more"
                     onClick={this.handleClick}
```

Comment:
Mmm. That breaks the theory of least surprise. Binding when you use it makes it look like other react examples. Otherwise use the autobind decorator.